.github/blunderbuss.yml

```diff
@@ -0,0 +1,6 @@
+assign_issues:
```

Comment:
Add `assign_prs:` with the same GitHub team so we have both issues and PRs auto-assigned.